src/app/auth.service.ts

```diff
@@ -28,4 +31,16 @@ export class AuthService {
   logout() {
     this.angularFireAuth.signOut();
   }
+
+  get User$(): Observable<User> {
```

Comment:
let's switch over to userObservable rather than the User$ to denote observables for readability